lib/cylc/cfgspec/suite.py

```diff
@@ -315,6 +318,16 @@ def _coerce_final_cycletime( value, keys, args ):
                 'execution timeout'           : vdr( vtype='interval_minutes'),
                 'reset timer'                 : vdr( vtype='boolean', default=False ),
                 },
+            'events' : {
+                'handler'                     : vdr( vtype='string_list', default=[] ),
+                'handler events'              : vdr( vtype='string_list', default=[] ),
+                'handler delays'              : vdr( vtype='interval_minutes_list', default=[] ),
```

Comment:
Hmm, I think maybe it should still be "handler retry delays", and just clearly document that the first interval applies to the first try?  Without "retry" it's not half as obvious what the delays are for.